packages/cli/package.json

```diff
@@ -68,7 +68,6 @@
     "@types/js-yaml": "^4.0.5",
     "@types/which": "^2.0.1",
     "fixturify": "^3.0.0",
-    "typescript": "^4.9.5",
```

Comment:
why are you removing typescript from devDeps here?

Reply:
One of tests does this. Already reverted.